src/site/filters/liquid.js

```diff
@@ -1086,4 +1087,19 @@ module.exports = function registerFilters() {
         return 'info';
     }
   };
+
+  liquid.filters.deriveLanguageTranslation = (lang, whichNode, id) => {
+    if (!lang) lang = 'en';
```

Comment:
I think you can avoid having to disable the `no-param-reassign` rule by assigning a default value to `lang`, like so:

```suggestion
  liquid.filters.deriveLanguageTranslation = (lang = 'en', whichNode, id) => {
```